### PR TITLE
chore(aim): Add Org State to unity routes

### PR DESCRIPTION
**Pull request by abshierjoel**
Adds an `OrganizationState` enum and the org state property to the endpoints that return orgs.

### CODE REVIEW

**Review by eatondustin1 — APPROVED** [review]
Nice, the enum schema definitely makes more sense, given that we need it for more endpoints than just the operator one.

File: src/unity/schemas/Organization.yml [hunk 1]
```diff
@@ -25,4 +25,7 @@ properties:
   clusterHost:
     type: string
     description: the url of the current cluster
+  state:
+    $ref: './OrganizationState.yml'
+    description: the current state of the organization
```
Comment (sanderson) [inline src/unity/schemas/Organization.yml:30]:
```suggestion
    description: URL of the current cluster
  state:
    $ref: './OrganizationState.yml'
    description: Current state of the organization
```

File: src/unity/schemas/OrganizationState.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+type: string
+description: state of an organization
```
Comment (sanderson) [inline src/unity/schemas/OrganizationState.yml:2]:
```suggestion
description: State of an organization
```

File: src/unity/schemas/OrganizationSummary.yml [hunk 3]
```diff
@@ -21,4 +21,7 @@ properties:
   regionName:
     type: string
     description: the friendly name of the region where the organization is hosted
+  state:
+    $ref: './OrganizationState.yml'
+    description: the current state of the organization
```
Comment (sanderson) [inline src/unity/schemas/OrganizationSummary.yml:26]:
```suggestion
    description: Current state of the organization
```

**Review by sanderson — APPROVED** [review]
Approved with minor suggestions

File: src/unity/schemas/OrganizationSummary.yml (hunk 3 above)
Comment (abshierjoel) [reply src/unity/schemas/OrganizationSummary.yml]: Thanks for the feedback! I got those changes added!

**Pull request merged**